app/controllers/v1/benchmarks_controller.rb

```diff
@@ -32,5 +32,11 @@ def resource
     def serializer
       BenchmarkSerializer
     end
+
+    def include
```

Comment:
This method does not seem to be called at all, does it? (Also indicated by missing coverage.)